src/live_restore/live_restore_server.c

```diff
@@ -105,7 +105,11 @@ __live_restore_worker_stop(WT_SESSION_IMPL *session, WT_THREAD *ctx)
     server->threads_working--;
 
     if (server->threads_working == 0) {
-        if (!F_ISSET(S2C(session), WT_CONN_CLOSING)) {
+        /*
+         * If the migration running flag isn't set it's because we're closing the connection. We
```

Comment:
```suggestion
         * If the migration running flag is set we expect to find work in the work queue. Otherwise, we're closing the connection.
```

Reply:
Reworded in [ced74dd](https://github.com/wiredtiger/wiredtiger/pull/11797/commits/ced74dd990c31cc2b8df2d47e2da537b686c648b) per our Slack discussion